Configurations/windows-makefile.tmpl

```diff
@@ -360,10 +360,12 @@ clean: libclean
 	{- join("\n\t", map { "-del /Q /F $_" } @PROGRAMS) -}
 	-del /Q /F $(ENGINES)
 	-del /Q /F $(SCRIPTS)
+	-del /Q /F $(GENERATED_MANDATORY)
```

Comment:
Please add the same in `Configurations/unix-Makefile.tmpl` as well.  If you dare, you can also add this line in `Configurations/descrip.mms.tmpl`:

``` make
        {- join("\n\t", map { "- DELETE $_;*" } @{$unified_info{depends}->{""}}) || "@ !" -}
```

Reply:
It was done for for unix-Makefile already under cygwin disguise.

Reply:
done.